src/App.js

```diff
@@ -1,24 +1,72 @@
-import React from 'react';
-import logo from './logo.svg';
-import './App.css';
+import React from "react";
+import logo from "./logo.svg";
```

Comment:
This is atavism.
In our project we do not use this logo.
Remove logo.svg and remove import